minimal/chipflow.toml

```diff
@@ -5,42 +5,19 @@ project_name = "chipflow-examples-minimal"
 soc = "design.design:MySoC"
 
 [chipflow.steps]
-sim = "design.steps.sim:MySimStep"
-board = "design.steps.board:MyBoardStep"
-silicon = "chipflow_lib.steps.silicon:SiliconStep"
 software = "design.steps.software:MySoftwareStep"
 
-[chipflow.clocks]
-default = 'sys_clk'
-
-[chipflow.resets]
-default = 'sys_rst_n'
-
 [chipflow.silicon]
 process = "ihp_sg13g2"
 package = "pga144"
 
-[chipflow.silicon.pads]
-# System
-sys_clk   = { type = "clock", loc = "114" }
-sys_rst_n = { type = "reset", loc = "115" }
+[chipflow.power.domains]
```

Comment:
I don't understand this. afaik IHP130 should be 1.2V or 1.5V core and 3.3V IO; I can't find any 5V IO libraries at all so I don't know why that's even being allowed